generators/server/templates/src/main/resources/config/_application.yml

```diff
@@ -238,7 +238,7 @@ server:
 # Properties to be exposed on the /info management endpoint
 info:
     # Comma separated list of profiles that will trigger the ribbon to show
-    display-ribbon-on-profiles: "dev"
+    displayRibbonOnProfiles: "dev"
```

Comment:
We use kebab-case for properties everywhere. Why changing it here ?

Reply:
@cbornet Because this will be part of the returned json and we need to be consistent, see @deepu105 comment : https://github.com/jhipster/generator-jhipster/pull/7050#discussion_r164187261

Reply:
Bringing inconsistency in the property file to get consistency in the JSON doesn't feel right...

Reply:
@cbornet Yes this is the behavior of Spring Boot but you are right all things considered I would also rather have a consistent yaml and an inconsistent json. So I will revert to `display-ribbon-on-profiles`.

Reply:
Maybe camelise the kebab strings in code ?

Reply:
Oh that’s now Spring Boot code ! So you can’t do much about it...

Reply:
ok then